packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -130,6 +141,55 @@ export default class LocusInfo {
     // update which required to compare different objects from locus
   }
 
+
+  getLocusPartner(participants, self) {
+    if (!participants || participants.length === 0) {
+      console.log('NO participants found ', participants);
+      return null;
+    }
+    return participants.find((participant) =>
+      (self && participant.identity !== self.identity)
+  && (participants.length <= 2 || (participant.type === USER && !participant.removed))) || this.partner;
+  }
+
+
+  isMeetingActive() {
```

Comment:
we should add a proxy for this to meeting/index

Reply:
We shouldnt depend on the proxy for now because any state change local or remote should trigger the state to ENDED or LEFT